pages/1.11/installing-upgrading/cloud/installing-aws/basic/ent/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout.pug
+title: AWS Basic Installation for DC/OS Enterprise
+navigationTitle: AWS Basic Installation for DC/OS Enterprise
+menuWeight: 10
+excerpt:
+
+enterprise: true
+---
+
+You can create a DC/OS cluster for Amazon Web Services (AWS) by using the DC/OS templates on AWS CloudFormation.
+
+These instructions provide a basic AWS CloudFormation template that creates a DC/OS cluster that is suitable for demonstrations and POCs. This is the fastest way to get started with the DC/OS templates for AWS CloudFormation.
+
+For a complete set of DC/OS configuration options, see the [Advanced AWS Install Guide](/1.11/installing-upgrading/cloud/installing-aws/advanced/ent/).
+
+**Important:** Upgrades are not supported with this installation method.
```

Comment:
Format this as a warning

Reply:
Content of ENT and OSS is merged in Advanced Installer page.